tests/contrib/pytest/test_pytest.py

```diff
@@ -120,6 +121,11 @@ def test_1(self, abc):
 
         assert len(spans) == 5
 
+        params = [1, 2, 3, 4, 5]
```

Comment:
Can we add a test with custom objects used in the parametrization?

Reply:
👍🏻